rpcserver.go

```diff
@@ -4158,6 +4159,16 @@ func createRPCOpenChannel(r *rpcServer, dbChannel *channeldb.OpenChannel,
 		RemoteChanReserveSat: int64(dbChannel.RemoteChanCfg.ChanReserve),
 	}
 
+	// Look up our channel peer's node alias if the caller requests it.
+	if peerAliasLookup {
+		peerAlias, err := r.server.graphDB.LookupAlias(nodePub)
+		if err != nil {
+			return nil, fmt.Errorf("unable to lookup peer "+
+				"alias: %v", err)
```

Comment:
nit: I think we want to start wrapping errors in new code (`%w` instead of `%v` for errors).